### PR TITLE
export screenSize and isMiniTablet

### PR DESCRIPTION
**Pull request by rohanhc**
Using exported values from xdpi and ydpi to give screenSize and isMiniTablet.

### CODE REVIEW

File: src/device.js [hunk 1]
```diff
@@ -160,6 +172,12 @@ if (exports.isMobile) {
     // Android then it's assumed to be an iOS device
     exports.isIOS = true;
   }
+
+  if (exports.isTablet) {
```
Comment (jishnu7) [inline src/device.js]: you can put this outside isAndroid check.

isMiniTablet should work irrespective of the platform.

File: src/device.js [hunk 2]
```diff
@@ -160,6 +172,12 @@ if (exports.isMobile) {
...
+    exports.isMiniTablet = screen_inches < 8;
+  } else {
+    exports.isMiniTablet = false;
+  }
```
Comment (rampr) [inline src/device.js]: this will work for iOS tablets too ?

File: src/device.js [hunk 3]
```diff
@@ -50,6 +50,16 @@ exports.registerDevice = function (name, path) {
   _devices[name] = path;
 };
 
+var xdpi = navigator.displayMetrics.xdpi,
+  ydpi = navigator.displayMetrics.ydpi,
```
Comment (rampr) [inline src/device.js]: won't this break on iOS ?

Comment (rohanhc) [inline src/device.js]: It won't, navigator.displayMetrics exists in ios as of now. I added a check at line 57.
I can work on a way to export from ios as well.

File: src/device.js (hunk 1 above)
Comment (rohanhc) [inline src/device.js]: It is outside isAndroid check

**Comment by jishnu7** [comment]
:+1:

File: src/device.js (hunk 3 above)
Comment (rampr) [inline src/device.js]: Sounds good, looks good to me now

**Comment by rampr** [comment]
Approved 👍

**Pull request merged**